scripts/format.sh

```diff
@@ -25,7 +25,7 @@ find src valhalla test bench -type f -name '*.h' -o -name '*.cc' \
 
 # Python setup
 py=$(setup_python)
-${py} -m pip install black==22.10.0 flake8==5.0.4
+${py} -m pip install --break-system-packages black==22.10.0 flake8==5.0.4
```

Comment:
newer pip (I think) has adopted an annoying parenting attitude and wouldn't let people install packages to system python anymore, even sudo doesn't help.. needs this flag. still better than requiring people to have a venv.

Reply:
will remove this again, now that we have another solution in master